### ISSUE TITLE
Program the AIS transponder with MMSI and other details

### ISSUE THREAD

**Issue by smr547** [CanberraOceanRacingClub/namadgi3#219]
As per issue #56 @mrmrmartin has obtained 503058550 as the MMSI for Namadgi 3. This MMSI must now be configured into the [VesperMarine AIS transponder](https://www2.vespermarine.com/XB6000-navstation-package.html).

The procedure for doing this is described on the *Configuration* page of the [installation guide](http://corc.com.au/Manuals/B46%20AIS%20Transponder%20Vesper%20XB6000-8000.pdf)

Note that additional equipment is required to perform the programming

* Laptop or Mac
* the WatchMate App* or the vmAIS PC/Mac
software. These are available for download from www.vespermarine.com/xb.
* a 1m long USB cable is required and some software (mini)

According to the [XB-6000 specification page](https://www2.vespermarine.com/xb6000-ais-transponder.html/#specifications), The AIS unit as a ``mini`` USB connector (there is a whole [range of USB sockets](https://en.wikipedia.org/wiki/USB#/media/File:USB_konektory.png)). The person doing the configuration (@mrmrmartin has volunteered) will have to bring the correct cable with them.

@jphayden and @DeeTaylor87 will be interested in this.

**Comment by DeeTaylor87** [comment]
Gosh. Good on you technical people for taking care of that!!!!!!!!

**Comment by smr547** [comment]
Yes, this one is a bit of a pain. I'm doing the research for @mrmrmartin so hopefully he will be able to program the unit on Saturday. The most likely failure point is the USB cable. :D

**Comment by DeeTaylor87** [comment]
Is it just the little mini HDMI cable that we use to charge phones?

**Comment by smr547** [comment]
``mini USB`` -- yes, those you use with phones -- should not be hard to find

More precisely, I think it is a ``mini A USB``  rather than a ``mini B USB`` (gotta love technology)

It also turns out that if we have an XB-8000 model, then the configuration can be done over WIFI

@mrmrmartin -- Hi Michael, I suggest you go into this issue and hit the ``subscribe`` button. You will then get immideate notifications as we collect this useful information for you.  :D

**Comment by smr547** [comment]
It seems that you will have to install [this software](https://www2.vespermarine.com/download.php?type=util&file=vmais&platform=win32) onto your windows laptop.

The installations procedures are fully described in the [xb8000 installation and config webpage](https://www2.vespermarine.com/guide/xb8000/)

While the XB8000 configuration can be done via wi-fi, I'd prefer us to use the USB cable. Can you find the USB cable that came with device (it may have been left on board by the installer). Otherwise you may have to source one tomorrow.

**Comment by DeeTaylor87** [comment]
MMSI number entered into AIS.  @smr547 Can you confirm that we are visible to other vessels/ users?

**Comment by smr547** [comment]
Namadgi 3's AIS transmissions were visible on web-based AIS monitoring systems throughout the passage south.  The iPad app ``FindShip`` seemed to have the best coverage. ``MarineTraffic.com`` did not seem to have full reception around Jervis Bay, however this site did report Namadgi 3 as ``Yacht`` on a number of occasions while also indicating that the AIS position report had been received via satellite. This bode's well for possible future global coverage via *satellite AIS*.

In short, the AIS transponder appears to be working well. Closing this item.
